src/processes/process.rs

```diff
@@ -45,6 +45,12 @@ impl PriorityClass {
     }
 }
 
+#[derive(Debug, Clone)]
+pub enum ProcessIdentifier {
+    Name(String),
+    PID(u32),
```

Comment:
Enums use `UpperCamelCase`. 
```suggestion
    Pid(u32),
```